src/Symfony/Component/Notifier/Bridge/Telegram/composer.json

```diff
@@ -20,6 +20,9 @@
         "symfony/http-client": "^4.3|^5.0",
         "symfony/notifier": "~5.0.0"
     },
+    "require-dev": {
+        "symfony/event-dispatcher": "^4.3|^5.0"
```

Comment:
Added because the `AbstractTransport` uses it in it's constructor. Not sure if it should actually be in `require`?